api/controllers/app.js

```diff
@@ -37,10 +37,11 @@ function canedit(user, rec) {
  * @apiSuccess {Object}         List of apps (maybe limited / skipped) and total count
  */
 router.get('/', common.jwt({credentialsRequired: false}), (req, res, next)=>{
-    var skip = req.query.skip||0;
+    let skip = req.query.skip||0;
     let limit = req.query.limit||100;
-    var ands = [];
+    let ands = [];
     if(req.query.find) ands.push(JSON.parse(req.query.find));
+    const listIncompatible = req.query.listIncompatible || false;
```

Comment:
A nomenclature issue for APIs, `list` can mean to return only incompatible apps. The endpoint also implicitly list apps, so seems redundant to tell it to list something.

A preferred verb would be `include`.